### ISSUE TITLE
Warning: componentWillUpdate has been renamed

### ISSUE THREAD

**Issue by hsh2001** [disqus/disqus-react#59]
### Describe the bug  
Error when using component.

### To Reproduce  
  1. Render `Disqus.DiscussionEmbed` component.

### Expected behavior  
No errors!

### Screenshots  
<img width="430" alt="스크린샷 2020-01-28 오후 1 48 03" src="https://user-images.githubusercontent.com/32220765/73236189-cf98ef00-41d4-11ea-8839-97f33c41e367.png">

### Specifications:  
 - OS: `macOS 10.15.2`
 - Package version: `^1.0.7`
 - Node version: `v12.14.0`

### Additional context  
When I use component, a warning appears. I think it need to update.

```
Warning: componentWillUpdate has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

* Move data fetching code or side effects to componentDidUpdate.
* Rename componentWillUpdate to UNSAFE_componentWillUpdate to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: DiscussionEmbed
```

**Comment by tterb** [comment]
@hsh2001 It looks like this was already fixed in #43, but the changes just haven't been published to npm yet.
I'll try to get an update published with these changes in the next couple days.

**Comment by tterb** [comment]
[v1.0.8](https://www.npmjs.com/package/disqus-react) has been published to npm
